libraries/opensk/src/api/crypto/mod.rs

```diff
@@ -0,0 +1,34 @@
+// Copyright 2023 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+pub mod ecdh;
+pub mod ecdsa;
+
+use self::ecdh::Ecdh;
+use self::ecdsa::Ecdsa;
+
+/// The size of field elements in the elliptic curve P256.
+pub const EC_FIELD_BYTE_SIZE: usize = 32;
+
+/// The size of a serialized ECDSA signature.
+pub const EC_SIGNATURE_SIZE: usize = 2 * EC_FIELD_BYTE_SIZE;
```

Comment:
I reused the names from Rust Crypto where I found a direct equivalent:

https://docs.rs/elliptic-curve/0.13.1/elliptic_curve/type.FieldBytesSize.html

The `EC_` prefix is because all constants are in the crypto-level mod, and I wanted to make it more obvious this is not AES etc. They are shared between ECDH and ECDSA.

Reply:
But it's the same, it's about coherence. I actually went to see if they have an issue about it and they do: https://github.com/RustCrypto/traits/issues/656. `FieldBytesSize` should be `FieldSize`. This suggests that we should use `EC_FIELD_SIZE` and `EC_SIGNATURE_SIZE`.

Also note that if you really wanted to use the same names it should have been `EC_FIELD_BYTES_SIZE`, because it's the size of `FIELD_BYTES`.

Reply:
Renamed to `EC_FIELD_SIZE`.